client/src/components/RegistrationForm/RegistrationForm.css

```diff
@@ -0,0 +1,30 @@
+.registration-form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 10px auto;
+    padding: 24px;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    max-width: 400px;
+    justify-content: center;
+    gap: 16px;
+    box-shadow: -13px 9px 50px -13px rgba(0, 0, 0, 0.75);
+}
+
+.additional-buttons button {
+    color: #000;
+}
+
+.additional-buttons .google-button {
```

Comment:
Use .scss extention for this file instead of .css.
Thus you can use nesting like this:
.additional-buttons {
  .google-button { ... }
}